libwidgets/View/SearchResults.vala

```diff
@@ -915,12 +915,13 @@ namespace Marlin.View.Chrome {
             return false;
         }
 
-        string ATTRIBUTES = FileAttribute.STANDARD_NAME + "," +
```

Comment:
The previous one with:
```
        const string ATTRIBUTES = FileAttribute.STANDARD_NAME + "," +
                            FileAttribute.STANDARD_DISPLAY_NAME + "," +
                            FileAttribute.STANDARD_CONTENT_TYPE + "," +
                            FileAttribute.STANDARD_IS_HIDDEN + "," +
                            FileAttribute.STANDARD_TYPE + "," +
                            FileAttribute.STANDARD_ICON;
```
was more memory efficient

Reply:
OK, reverted.